README.md

```diff
@@ -74,24 +84,31 @@ $ /usr/local/bin/master-restart controller
 On Openshift 4 cluster resources are modified by the operator if the controller is installed there and you can skip these steps. If you chose not to install the controller on your Openshift 4 cluster you will need to perform these steps manually.
 
 If you haven't already, determine the CORS URL that needs to be added retrieve the route URL
-`oc get -n openshift-migration route/migration -o go-template='{{ .spec.host }}{{ println }}'`
+`oc get -n openshift-migration route/migration -o go-template='(?i}//{{ .spec.host }}(:|\z){{ println }}' | sed 's,\.,\\.,g''`
+
+#### For Openshift 4.2
+`oc edit apiserver cluster` and ensure the following exist:
```

Comment:
I might add a note here that it's expected that the operator will correctly configure this value, but it may take some non-zero amount of time.